File: Lecture003 If-Else, While loop & Lots of Patterns/conditionals.java
import java.util.Scanner;

class conditionals {
    public static void main(String[] args) {
        int a;

        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the value of a: ");

        a = sc.nextInt();

        sc.close();

        if (a > 0) {
            System.out.println("A is positive.");
        } else if (a < 0) {
            System.out.println("A is negative.");
        } else {
            System.out.println("A is zero.");
        }
    }
}

File: Lecture003 If-Else, While loop & Lots of Patterns/conditionals.js
// If you want to take JavaScript input into the browser
/*
let a = parseInt(prompt("Enter the value of a"));
if (a > 0) {
  console.log("a is positive");
} else if (a < 0) {
  console.log("a is negative");
} else {
  console.log("a is 0");
}
*/

// If you want to take input on the JavaScript command line
const readline = require("readline").createInterface({
  input: process.stdin,
  output: process.stdout,
});

readline.question("Enter the value of a: ", (input) => {
  const a = parseInt(input);
  if (a > 0) {
    console.log("a is positive");
  } else if (a < 0) {
    console.log("a is negative");
  } else {
    console.log("a is 0");
  }
  readline.close();
});